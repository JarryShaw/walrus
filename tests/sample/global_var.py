a = 6
print(b := a * a)
print(b + 1)

for _ in range(2):
    c = 1
    print(c := 2)


for x in (6, 0):
    if a and (d := x):
        print(d)
    elif e := 7:
        print(e)
    else:
        print('Nope :=')
