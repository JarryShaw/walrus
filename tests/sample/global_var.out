36
37
2
2
6
7
